rllib/models/modelv2.py

```diff
@@ -66,6 +78,7 @@ def get_initial_state(self):
         """
         return []
 
+    @PublicAPI
     def forward(self, input_dict, state, seq_lens):
```

Comment:
We have been inconsistent about whether to mark methods as public api or not, if the entire class is PublicAPI (I believe currently usually not). Should we favor one or the other?

I don't think the current documentation for PublicAPI mentions this.

Reply:
Yes, we should favor marking the particular methods as well. I think that's more explicit. I would even say we do this in the child classes (and their methods). It's confusing otherwise.